LayoutTests/media/modern-media-controls/ios-inline-media-controls/touch/ios-inline-media-controls-timeline-scrub-stops-propagation.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <script src="../../../../resources/js-test-pre.js"></script>
+    <script src="../../resources/media-controls-utils.js"></script>
+    <script src="../../../../resources/ui-helper.js"></script>
+</head>
+<body>
+    <video id="video" src="../../../content/test.mp4" controls style="width: 400px;"></video>
+    <script>
+        window.jsTestIsAsync = true;
+        
+        description("Test that scrubbing the timeline does not propagate touch events to parent elements.");
+        
+        const video = document.getElementById('video');
+        let containerReceivedTouchStart = false;
+        let containerReceivedTouchMove = false;
+        let containerReceivedTouchEnd = false;
+        
+        video.addEventListener('touchstart', () => { containerReceivedTouchStart = true; }, false);
+        video.addEventListener('touchmove', () => { containerReceivedTouchMove = true; }, false);
+        video.addEventListener('touchend', () => { containerReceivedTouchEnd = true; }, false);
+        
+        video.addEventListener('loadedmetadata', async () => {
+            await new Promise(resolve => setTimeout(resolve, 500));
```

Comment:
Let's not add an unnecessary 500ms wait here.  Look at `testExpectedEventuallySilent()` from video-test.js; if we need to wait for the shadow root to be created, we can use that function to loop until the shadow root is available.